src/nvim/main.c

```diff
@@ -927,9 +926,6 @@ static void parse_command_name(mparm_T *parmp)
   if (parse_char_i(&initstr, 'r'))
```

Comment:
@Pyrohh @fmoralesc I assume line 906 was moved after the rebase, so this comment was addressed?